src/postgres/postgresConnectionStrings.ts

```diff
@@ -27,13 +28,18 @@ export class ParsedPostgresConnectionString extends ParsedConnectionString {
     public username: string | undefined;
     public password: string | undefined;
     public readonly port: string;
+    public readonly ssl: boolean | ConnectionOptions | undefined;
```

Comment:
Do we still need these ssl changes? Since the azure scenario always uses `BaltimoreCyberTrustRoot` and the attached scenario always uses the connectionString itself

Reply:
> Make sure to test out the localhost scenario - I'm still seeing "Must enter credentials" when I should be seeing the databases _without_ needing to enter credentials
> <img alt="Screen Shot 2020-08-26 at 10 37 45 AM" width="378" src="https://user-images.githubusercontent.com/11282622/91337504-4c4e7480-e788-11ea-8011-003a3796a8c6.png">

I am able to connect, you just have to include the username in the connection string for it to work.

Reply:
That's true, we don't need ssl field.